cli/smartcontract/smart_contract.go

```diff
@@ -435,6 +452,10 @@ func invokeInternal(ctx *cli.Context, withMethod bool, signAndPush bool) error {
 	}
 	if len(args) > paramsStart {
 		for k, s := range args[paramsStart:] {
+			if s == hashesForVerifyingSeparator {
```

Comment:
This should depend on `signAndPush` to not make this functionality available to real invocations.